test/integration/lib/foxboxHelper.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+var helper = (function() {
+
+    function removeDB() {
+      var fs = require('fs');
+      var filePath = process.env.HOME + 
+      '/.local/share/foxbox/users_db.sqlite' ; 
+      fs.exists(filePath,function(exists){
+        if(exists) {
+          fs.unlinkSync(filePath);
```

Comment:
I thought about it, but actually, the main needs to be blocked in order to proceed to the next test.  IF the db is not removed before moving on to the next test, the test would fail.